return {

	InGame = { -- Arsenal
		['286090429'] = {
			Title    = 'Arsual v.52',
			Author   = 'by fr¿eze08',
			Callback =function()
				local sus,err = pcall(function()
					loadstring(game:HttpGet(('https://raw.githubusercontent.com/usernil01101/JGS-SDD/refs/heads/main/cache%20/Arsual.luau'),true))()
				end)
				

			end,
		},

	},
	
	Universal = {
		['Infinite Yield'] = {
			Title    = 'Infinite',
			Author   = 'by Unknown author',
			Callback =function()
				loadstring(game:HttpGet(('https://raw.githubusercontent.com/EdgeIY/infiniteyield/master/source'),true))()
				setclipboard('https://raw.githubusercontent.com/EdgeIY/infiniteyield/master/source')
			end,
		},
	},
		
	

}
